### PR TITLE
Fix broken E2E test

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

**Review by AlexandreBrown — APPROVED** [review]

**Comment by ArangoGutierrez** [comment]
Umm unsure what failed here ... 🤔

**Comment by alculquicondor** [comment]
uhm... did you try locally?
Or it could be one of those rare scenarios where coredns took too long to set the DNS entries?

**Comment by ArangoGutierrez** [comment]
> 

yup, `make test-images` worked here 
1_DPS45Tufsih-zED3To4k6g

File: Makefile [hunk 1]
```diff
@@ -96,7 +96,7 @@ test_images:
 	${IMG_BUILDER} build -t mpioperator/mpi-pi:openmpi examples/v2beta1/pi
 	${IMG_BUILDER} build -t mpioperator/intel build/base -f build/base/intel.Dockerfile
 	${IMG_BUILDER} build -t mpioperator/intel-builder build/base -f build/base/intel-builder.Dockerfile
-	${IMG_BUILDER} build -t mpioperator/mpi-pi:intel examples/v2beta1/pi -f examples/pi/intel.Dockerfile
+	${IMG_BUILDER} build -t mpioperator/mpi-pi:intel build/base -f build/base/intel.Dockerfile
```
Comment (alculquicondor) [inline Makefile:99]:
```suggestion
	${IMG_BUILDER} build -t examples/v2beta1/pi -f examples/v2beta1/pi/intel.Dockerfile
```

Comment (alculquicondor) [reply Makefile:99]: good we have E2E tests :)

Comment (ArangoGutierrez) [reply Makefile:99]: tag the image as `-t examples/v2beta1/pi` ?

Comment (alculquicondor) [reply Makefile:99]: Oops, deleted too much

Comment (alculquicondor) [reply Makefile]: You applied the old edit :(

**Comment by alculquicondor** [comment]
Do we need to re-trigger the tests even after a new commit was pushed?

**Comment by ArangoGutierrez** [comment]
yay! working!

**Review by terrytangyuan — APPROVED** [review]
Thanks!
/lgtm
/approve

**Pull request merged**